tasks/main.yml

```diff
@@ -1,6 +1,8 @@
 # tasks file for r
 ---
 - include: repository.yml
+  when:
+    - r_cran_repo == true
```

Comment:
You should remove the '== true' on this line to have the a working build on travis
Something like : 
```
- include: repository.yml
  when:
    - r_cran_repo
```
As it is a boolean

Reply:
You are right. Or maybe `r_cran_repo | bool` just to be safe